### PR TITLE
fix: Prevent error and incorrect duration

### PR DESCRIPTION
**Pull request by cuyl**
The duration is the minimum of the end times of all active streams.
Non-active streams are not guaranteed to have useful maxTimestamp
values, due to the lazy-loading system, so they are ignored.

### CODE REVIEW

File: lib/hls/hls_parser.js [hunk 1]
```diff
@@ -308,7 +308,13 @@ shaka.hls.HlsParser = class {
       const PresentationType = shaka.hls.HlsParser.PresentationType_;
       this.setPresentationType_(PresentationType.VOD);
 
-      const maxTimestamps = streamInfos.map((s) => s.maxTimestamp);
+      // Only check active stream:
```
Comment (theodab) [inline lib/hls/hls_parser.js:311]: I am pretty sure this change is correct. It's actually a change I had been considering, while diagnosing an issue that wasn't filed on GitHub.

However, this block of comments is hard to parse, grammatically. And honestly, we probably don't need to go into this much detail.
I'd rephrase like this:
```js
      // The duration is the minimum of the end times of all active streams.
      // Non-active streams are not guaranteed to have useful maxTimestamp
      // values, due to the lazy-loading system, so they are ignored.
```

**Review by theodab — CHANGES_REQUESTED** [review]

File: lib/hls/hls_parser.js (hunk 1 above)
Comment (cuyl) [reply lib/hls/hls_parser.js]: Thanks.

**Comment by cuyl** [comment]
@theodab  I appreciate your suggestions and am sorry for the poor expression.

**Review by theodab — APPROVED** [review]

**Comment by theodab** [comment]
Thank you for your contribution!
